types/src/transaction/change_set.rs

```diff
@@ -6,6 +6,41 @@ use crate::{contract_event::ContractEvent, write_set::WriteSet};
 use move_core_types::vm_status::{StatusCode, VMStatus};
 use serde::{Deserialize, Serialize};
 
+#[derive(Clone, Copy)]
+pub struct ChangeSetLimits {
+    pub max_bytes_per_write_op: u64,
+    pub max_bytes_all_write_ops_per_transaction: u64,
+    pub max_bytes_per_event: u64,
+    pub max_bytes_all_events_per_transaction: u64,
+    pub creation_as_modification: bool,
```

Comment:
This is not limit, maybe called `ChangeSetGasConfigs`

Reply:
? this is to address magic number 3 used below.

Reply:
anything reasonable but not limits works for me as I know you are paranoid.